text/pdf_highlight_text.go

```diff
@@ -0,0 +1,135 @@
+/*
+ * Highlight text: Hightlight target texts inside the PDF file.
+ *
+ * Run as: go run pdf_highlight_text.go inputFile.pdf outputFile.pdf term
+ */
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/unidoc/unipdf/v3/common/license"
+	"github.com/unidoc/unipdf/v3/creator"
+	"github.com/unidoc/unipdf/v3/extractor"
+	"github.com/unidoc/unipdf/v3/model"
+)
+
+func init() {
+	err := license.SetMeteredKey(`put your license key here`)
+	if err != nil {
+		panic(err)
+	}
+}
+func main() {
+	if len(os.Args) < 4 {
+		fmt.Printf("Usage: go run pdf_highlight_text.go <input.pdf> <output.pdf> <term> \n")
+		os.Exit(0)
+	}
+	inputPath := os.Args[1]
+	outputPath := os.Args[2]
+	term := os.Args[3]
+
+	err := hightlightWords(inputPath, outputPath, term)
+	if err != nil {
+		panic(err)
+	}
+	fmt.Printf("Successfully highlighted the word %s and created %s\n", term, outputPath)
+}
```

Comment:
should format with goimports

Reply:
add a comment like
getBoundingBoxes returns the bounding boxes of all instances of the search term.